src/index.ts

```diff
@@ -20,36 +29,49 @@ interface IIndex {
 export class Table {
   public static transactWrite(transactions: any, options = { verbose: false }) {
     const { verbose } = options;
-    delete options.verbose;
     return transactWrite({ docClient, transactions, options, verbose });
   }
 
-  public static replaceDynamoClient(newDynamodb, newClient) {
-    dynamodb = newDynamodb;
-    docClient = newClient;
+  public static replaceDynamoClient(newDbClient: DynamoDBClient, newDocClient: DynamoDBDocumentClient) {
+    dbClient = newDbClient;
+    docClient = newDocClient;
   }
 
   public tableName: string;
   public partitionKey: string | undefined;
   public sortKey: string | undefined;
   public indexMap: Map<string, IIndex>;
+  public dbClient: DynamoDBClient;
+  public docClient: DynamoDBDocumentClient;
   private initialized: boolean;
 
-  constructor(name: string) {
+  constructor(name: string, config?: DynamoDBClientConfig) {
     this.tableName = name;
     this.initialized = false;
     this.partitionKey = undefined;
     this.sortKey = undefined;
     this.indexMap = new Map();
+    if (config) {
+      this.dbClient = new DynamoDBClient(config);
+      this.docClient = DynamoDBDocumentClient.from(this.dbClient);
+    } else {
+      this.dbClient = dbClient;
+      this.docClient = docClient;
+    }
   }
 
-  public async initTable() {
-    const tableInfo: any = await dynamodb.describeTable({ TableName: this.tableName }).promise();
+  public async initTable(): Promise<void> {
+    // const tableInfo: any = await dynamodb.describeTable({ TableName: this.tableName }).promise();
```

Comment:
```suggestion
```

Usually, we want to clean up any code-comments since we have the version control system (Git).

Reply:
Cleanup is done.